### PR TITLE
Let's not forget to change the fromSchema and toSchema functions

### PR DESCRIPTION
**Pull request by WilcoLouwerse**

### CODE REVIEW

**Review by rjzondervan — CHANGES_REQUESTED** [review]
The changes done here do not seem to reflect the changes that are actually made.

**Review by rjzondervan — APPROVED** [review]

File: api/src/Subscriber/ProxySubscriber.php [hunk 1]
```diff
@@ -74,7 +77,7 @@ public function proxy(RequestEvent $event): void
                 $method,
                 [
                     'headers' => $headers,
-                    'query'   => $event->getRequest()->query->all(),
+                    'query'   => $this->requestService->realRequestQueryAll($method, $event->getRequest()->getQueryString()),
```
Comment (bbrands02) [inline api/src/Subscriber/ProxySubscriber.php:80]: What does this function do different?

Comment (WilcoLouwerse) [reply api/src/Subscriber/ProxySubscriber.php:80]: Both options convert all query params to an array.
$event->getRequest()->query->all() (/Symfony)
Will change _ into . and 'break' stuff cause it is incorrect / not the actual query param
realRequestQueryAll() will convert query params to an array using the queryString, so it won't have this problem

Comment (WilcoLouwerse) [reply api/src/Subscriber/ProxySubscriber.php:80]: And everywhere else we do this we use realRequestQueryAll (for this reason)

**Pull request merged**